public/app/core/components/ForgottenPassword/ForgottenPassword.test.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { asyncScheduler, scheduled, throwError } from 'rxjs';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { ForgottenPassword, SendResetEmailDTO } from './ForgottenPassword';
+import { backendSrv } from '../../services/backend_srv';
+import { toAsyncOfResult } from '../../../features/query/state/DashboardQueryRunner/testHelpers';
+import { createFetchResponse } from '../../../../test/helpers/createFetchResponse';
+
+jest.mock('@grafana/runtime', () => ({
+  ...((jest.requireActual('@grafana/runtime') as unknown) as object),
+  getBackendSrv: () => backendSrv,
+}));
+
+function getTestContext() {
+  jest.clearAllMocks();
+  const fetchMock = jest.spyOn(backendSrv, 'fetch');
+  const { rerender } = render(<ForgottenPassword />);
+
+  return { fetchMock, rerender };
+}
+
+describe('ForgottenPassword', () => {
+  describe('when mounted', () => {
+    it('then it should show input field', () => {
+      getTestContext();
+
+      expect(screen.getByRole('textbox')).toBeInTheDocument();
+    });
+
+    it('then it should show send button', () => {
+      getTestContext();
+
+      expect(screen.getByText(/send reset email/i)).toBeInTheDocument();
```

Comment:
```suggestion
      expect(screen.getByRole('button', { name: /send reset email/i })).toBeInTheDocument();
```
`getByText` should be used as one of the last resorts, if we can't match the element by other queries. The benefit of using `getByRole` here, besides being more specific that it's a button and not just some text appearing on the screen, is that it also tests that the said button is accessible to screen readers.

Reply:
Fixed!